switchlink/switchlink_db.c

```diff
@@ -237,7 +237,7 @@ switchlink_db_status_t switchlink_db_interface_update(
  *    SWITCHLINK_DB_STATUS_ITEM_NOT_FOUND otherwise
  */
 
-switchlink_db_status_t switchlink_db_interface_delete(uint32_t ifindex) {
+switchlink_db_status_t switchlink_db_delete_interface(uint32_t ifindex) {
```

Comment:
Any reason why all the names has been modified?
Following any specific guidelines for function names?

Reply:
names were changed to be more readable. Guideline is to have a verb first